pkg/lib/patternfly/patternfly-4-overrides.scss

```diff
@@ -39,11 +39,31 @@ svg {
     }
 }
 
-// When using horizontal ruler inside description list it's just for the spacing - don't show it
+// Override the description list to use smaller spacing and subgrid (where available)
 .pf-c-description-list {
-    // The default gap between the rows is too large
-    --pf-c-description-list--RowGap: 1rem;
+    // Decrease the vertical spacing between lines
+    --pf-c-description-list--RowGap: var(--pf-global--spacer--sm);
```

Comment:
To me it seems this is too little spacing, especially because the space between the label and the value remains big.
Before:

![Screen Shot 2021-02-04 at 10 53 32](https://user-images.githubusercontent.com/14921356
/106875909-79a83280-66d7-11eb-92c9-aca2b06f0daa.png)

After:

![Screen Shot 2021-02-04 at 10 52 57](https://user-images.githubusercontent.com/14921356/106875912-7a40c900-66d7-11eb-964a-1de6f236c9b5.png)

And vertical lists also are too condensed I feel:

Before:

![Screen Shot 2021-02-04 at 10 57 47](https://user-images.githubusercontent.com/14921356/106876310-dd326000-66d7-11eb-9f92-eb274ef3e232.png)

After: 

![Screen Shot 2021-02-04 at 10 57 43](https://user-images.githubusercontent.com/14921356/106876314-ddcaf680-66d7-11eb-948a-5bde76314ca4.png)

Reply:
Yeah, I think this PR predates the vertical lists? (Or we just weren't using them?) Anyway, it was made with horizontal ones in mind.